test/support/route.go

```diff
@@ -36,3 +40,45 @@ func GetRoute(t Test, namespace, name string) *routev1.Route {
 	t.T().Helper()
 	return Route(t, namespace, name)(t)
 }
+
+func ExposeService(t Test, namespace string, service string, route string, port string, expetedCode int) *routev1.Route {
```

Comment:
The parameter names may be misleading, what about:
serviceName
servicePort
routeName